.eslintrc

```
@@ -6,8 +6,8 @@
       "ecmaVersion": 2020,
       "sourceType": "module"
     },
-    "extends": ["airbnb", "prettier"],
-    "plugins": ["react", "prettier"],
+    "extends": ["prettier"],
```

I recently discussed prettier and eslint configs with @EdvinasDaugirdas, and he recommended using the `eslint:recommended` base config and mentioned that's what they tend to default to in their team. We've been applying this to jslib-aws recently: https://github.com/grafana/k6-jslib-aws/pull/81/files#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822R3

Maybe we want to do the same here?

```suggestion
    "extends": ["eslint:recommended"],
```

---

@oleiade thanks! I've used that config, and the last commit contains the fixes required by this rule. Please check and let me know if they are fine :+1: